content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -166,18 +166,12 @@ Use the following commands to install Containerd on your system:
 1. Install containerd:
 
    Visit
-   [Getting started with containerd](https://containerd.io/docs/getting-started/#starting-containerd)
+   [Getting started with containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md)
    and follow the instructions there, up to the point where you have a valid
    configuration file (on Linux: `/etc/containerd/config.toml`).
```

Comment:
@marosset I feel like with the last statement we are making it confusing for a first-time user who might assume that config.toml is the config file for Linux OS only.
```suggestion
   configuration file, config.toml. On Linux, you can find this file under the path `/etc/containerd`.
```

Reply:
updated.